PWGMM/Lumi/Tasks/CMakeLists.txt

```diff
@@ -34,3 +34,12 @@ o2physics_add_dpl_workflow(fitvdm
         O2::DetectorsCommonDataFormats
         O2::DetectorsVertexing
         COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(lumistab
+        SOURCES lumiStability.cxx
+        PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
+        O2::ReconstructionDataFormats
```

Comment:
There is no need to link to libraries that are not used, please remove the extras. If or when you add code to this task that requires these libraries, you can add them.

Reply:
Alright, I will do the changes